server/admin_management/tests/constants.py

```diff
@@ -119,19 +125,26 @@
     }
 )
 TEST_ACCESS_CONTROL_PRIVILEGE_CREATE_REQUEST = {
-    "user_name": TEST_USER_NAME,
+    "user_name": TEST_USER_NAME_IDIR,
     "user_guid": TEST_USER_GUID_IDIR,
     "user_type_code": famConstants.UserType.IDIR,
     "role_id": TEST_FOM_DEV_SUBMITTER_ROLE_ID,
     "forest_client_numbers": [TEST_FOREST_CLIENT_NUMBER],
 }
 
 TEST_ACCESS_CONTROL_PRIVILEGE_CREATE_REQUEST_CONCRETE = {
-    "user_name": TEST_USER_NAME,
+    "user_name": TEST_USER_NAME_IDIR,
     "user_guid": TEST_USER_GUID_IDIR,
     "user_type_code": famConstants.UserType.IDIR,
     "role_id": TEST_FOM_DEV_REVIEWER_ROLE_ID,
 }
+TEST_ACP_CREATE_CONCRETE_BCEID = {
```

Comment:
Since you use "_ACP_" here, not sure if you also like to rename other variable above related to "_ACP_" with short comment on what "_ACP_" is?

Reply:
yeah I was thinking of that before, but if I made those changes in this pr, there are too many tests need to change, and I was only focus on the broken tests. But I'll refactor them when adding new tests later, in following prs

Reply:
I see, thanks, I didn't notice that, sure, we can update them later time.